tests/lib/managers/terms-of-service-test.js

```diff
@@ -0,0 +1,571 @@
+				});
+		});
+	});
+
+	describe('createUserStatuses()', function() {
+		it('should make POST request to create custom terms of service user status when called', function() {
+			var expectedParam = { body: { tos: {tos_id: '1234', type: 'terms_of_service'}, is_accepted: true} };
+
+			var response = {
+				type: 'terms_of_service_user_status',
+				id: '1234',
+				tos: {},
+				user: {},
+				is_accepted: true,
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('post').withArgs('/terms_of_service_user_statuses', expectedParam)
+				.returns(Promise.resolve(response));
+			termsOfService.createUserStatuses(TERMS_OF_SERVICE_ID, true, TERMS_OF_SERVICE_USER_ID);
+		});
+
+		it('should call callback with the created terms of service user status info when a 200 response is returned', function(done) {
+			var response = {
+				type: 'terms_of_service_user_status',
+				id: '1234',
+				tos: {},
+				user: {},
+				is_accepted: true,
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'post').yieldsAsync(null, response);
+			termsOfService.createUserStatuses(TERMS_OF_SERVICE_ID, true, TERMS_OF_SERVICE_USER_ID, function(err, termsOfServiceUserStatusObject) {
+				assert.ifError(err);
+				assert.strictEqual(termsOfServiceUserStatusObject, response, 'terms of service user object is returned');
+				done();
+			});
+		});
+
+		it('should return a promise resolving to created terms of service user status object when a 200 response is returned', function() {
+			var response = {
+				type: 'terms_of_service_user_status',
+				id: '1234',
+				tos: {},
+				user: {},
+				is_accepted: true,
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'post').returns(Promise.resolve(response));
+			return termsOfService.createUserStatuses(TERMS_OF_SERVICE_ID, true, TERMS_OF_SERVICE_USER_ID)
+				.then(termsOfServiceUserStatusObject => {
+					assert.strictEqual(termsOfServiceUserStatusObject, response, 'terms of service user status object is returned');
+				});
+		});
+	});
+
+	describe('getUserStatuses()', function() {
+		it('should make GET request to retrieve custom terms of service user status when called', function() {
+			var response = {
+				total_count: 2,
+				entries: []
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('get').withArgs('/terms_of_service_user_statuses')
+				.returns(Promise.resolve(response));
+			termsOfService.getUserStatuses(TERMS_OF_SERVICE_ID);
```

Comment:
You're passing an ID here but not verifying it anywhere else in the test — need to make sure it's being correctly used